Mu2eG4/src/Mu2eG4PerThreadStorage.cc

```diff
@@ -28,8 +28,8 @@ namespace mu2e {
     art::ProductID simPartId(evt->getProductID<SimParticleCollection>());
     art::EDProductGetter const* simProductGetter = evt->productGetter(simPartId);
 
-    simParticleHelper = SimParticleHelper(ioconf.inputs().simParticleNumberOffset(), simPartId, evt, simProductGetter);
-    simParticlePrimaryHelper = SimParticlePrimaryHelper(evt, simPartId, simProductGetter);
+    simParticleHelper.emplace(ioconf.inputs().simParticleNumberOffset(), simPartId, evt, simProductGetter);
```

Comment:
I thought particlenumberoffset was replaced with stage?  Or is this an internal calculation?

Reply:
see below